verifier/trustpolicy/trustpolicy.go

```diff
@@ -150,10 +151,13 @@ type SignatureVerification struct {
 	Override          map[ValidationType]ValidationAction `json:"override,omitempty"`
 }
 
-// Validate validates a policy document according to it's version's rule set.
+// Validate validates a policy document according to its version's rule set.
 // if any rule is violated, returns an error
 func (policyDoc *Document) Validate() error {
 	// Validate Version
+	if policyDoc.Version == "" {
```

Comment:
Can `policyDoc` be `nil`?

Reply:
It's checked before the Validate() method is called. Thus, in notation's workflow, it's safe. However, since it's a public method, applications that want to import the trustpolicy package and call Validate() directly shall check nil on their side. Otherwise, a panic will be triggered.

Reply:
Let's check it.